specs/global/responses.json

```diff
@@ -435,11 +435,11 @@
             }
         }
     },
-    "QualifierHash": {
+    "qualifier_hash": {
         "description": "Modified Qualifier Hash",
         "content": {
             "application/json": {
-                "schema": { "$ref": "./schemas.json#/QualifierHash" }
```

Comment:
I think this doesn't constitute as a parameter name, this is to describe some internal Schema